src/patternfly/components/Alert/examples/alert-danger-example.hbs

```diff
@@ -1,18 +1,22 @@
 {{#> alert alert--modifier="pf-m-danger" alert--attribute='aria-label="Danger Notification"'}}
   {{#> alert-icon alert-icon--danger="true"}}
   {{/alert-icon}}
-    {{#> alert-body alert-HasDescription="true"}}
-      {{#> alert-title}}
+  {{#> alert-body alert-body--attribute='role="alert"'}}
+    {{#> alert-title}}
       {{#> accessibility accessibility--type="sr-only"}}Danger: {{/accessibility}}
         Danger notification title
-      {{/alert-title}}
+    {{/alert-title}}
+    {{#> alert-description}}
+			Danger notification description.
+      <a>This is a link.</a>
```

Comment:
nitpicky, but can you fix the indenting, put the link on the same line as the sentence and add `href="#"` to the `<a>` element? (and do that for the other alert examples in the demo, too)

Reply:
Oh, good catch! I would have expected the aXe tooling to flag that. 😕

Reply:
yes fixed :)